src/utils/image_loader.py

```
@@ -0,0 +1,12 @@
+from PIL import Image
+import os
+
+def load_image(image_path):
+    if not os.path.exists(image_path):
+        raise FileNotFoundError(f"File {image_path} not found.")
+    
+    try:
+        image = Image.open(image_path)
+        return image
+    except Exception as e:
+        raise IOError(f"Error while loading file: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)